tox.ini

```diff
@@ -65,3 +65,4 @@ commands =
     docstrings: sphinx-autobuild -BqT -z lms -z tests -b dirhtml {envdir}/rst {envdir}/dirhtml
     checkdocstrings: sphinx-build -qTn -b dirhtml {envdir}/rst {envdir}/dirhtml
     clean: coverage erase
+sitepackages = {env:SITE_PACKAGES:false}
```

Comment:
Can you imagine a scenario where this leads to mismatched package versions? Seems safe, but worth a second thought.

Reply:
There's an increased risk but I think it's safe. Most of the tox commands would already be installing the same Python deps that are in sitepackages into the venv anyway,  if sitepackages were not on, so this shouldn't make a difference. _Some_ of the tox commands wouldn't be, for example backend linting doesn't actually need the sitepackages, but I think we'll be fine.